README.md

```diff
@@ -1,2 +1,84 @@
-# admission-controllers
-Admission Controllers used in LitmusChaos 
+# Litmus-Admission-Controllers
+
+Litmus Admission Webhook is an extension of LitmusChaos Chaos-Operator. This Webhook helps in validating the Chaos target specified in ChaosEngines.
+It will be enhanced for validating the secondary resources such as Application related details, RBAC's, ChaosExperiments, Annotations and much more.
+
+
+## Validation performed
+
+Currently this Webhook validates the application information provided in chaoengine(`.spec.appInfo`).
+    - It validates if the resource searched for exists in the k8s cluster or not.
+
+
+## Installation Steps:
+
+- Using current `litmus` ServiceAccount provided for chaos-operator (https://docs.litmuschaos.io/docs/getstarted/#install-litmus) will work for this deployment, or else a ServiceAccount linked with a ClusterRole with the following permission will work.
+Prefer this deployment to run in parallel with chaos-operator.
+
+```
+rules:
+  - apiGroups: ["","apps","litmuschaos.io"]
+    resources: ["pods","jobs","deployments","replicationcontrollers","daemonsets","replicasets","statefulsets","chaosengines"]
+    verbs: ["get","create","update","patch","delete","list","watch","deletecollection"]
+  - apiGroups: ["admissionregistration.k8s.io"]
+    resources: ["validatingwebhookconfigurations"]
+    verbs: ["get","create","list","delete","update"]
+```
+ 
```

Comment:
Let us not describe this. Instead just state that this uses the same serviceAccount as the operator (we are anyways ensuring via PRs to pages and helm repos to ensure that the rbac is updated with the below)

Reply:
@ksatchit should I create a PR to add that in litmus-helm?

Reply:
Yes. It is good to add these to the helm (the permissions)..though I still don't plan the admission-controller deployment being packaged into our templates yet as it is alpha. The pages repo has a [PR](https://github.com/litmuschaos/pages/pull/46) which includes this permissions into the 1.4 integrated-operator yaml, so, yes, makes sense to include into helm deployments as well.